pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -927,3 +982,13 @@ func mergeMaps(m1, m2 map[string]string) map[string]string {
 	}
 	return merged
 }
+
+func defaultObservabilityConfigMap() *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{Name: metrics.ConfigMapName()},
+		Data: map[string]string{
+			//TODO: Better nonempty config
+			"_example": "See tekton-pipelines namespace for valid values",
```

Comment:
So, one thing here is that this _is_ a sensible default in that it will automatically select a prometheus exporter on 9090. It just needs to have _one key_ so that the configmap watcher things the configmap is valid. See what we do for this in the install: https://github.com/tektoncd/triggers/blob/main/config/config-observability.yaml#L24

I could just copy the example value whole hog, but I didn't want to duplicate code too much.

Reply:
ok, I'd change the message to say See config-observability in tekton-pipelines (or a link to the docs) for valid values